src/routes/trip.route.js

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+import TripController from '../controllers/trip.controller';
+import { validateReturnTrip } from '../validations/trip.validation';
+import protectRoute from '../middlewares/protect-route.middleware';
+
+const router = Router();
+
+router.post('/return_trip', validateReturnTrip, protectRoute, TripController.returnTripController);
```

Comment:
Before continuing to validate input you should pass from protectRoute to validateReturnTrip